### PR TITLE
ISPN-6801 TransactionXaAdapter and SynchronizationAdapter are too big

### PR DESCRIPTION
**Pull request by danberindei**
https://issues.jboss.org/browse/ISPN-6801

Move bulk of synchronization/xa adapters to TransactionTable

### CODE REVIEW

**Comment by ryanemerson** [comment]
@danberindei Looks good to me, but I'll hold of merging until somebody more familiar with the code has had a look.

File: core/src/main/java/org/infinispan/transaction/impl/AbstractEnlistmentAdapter.java [hunk 1]
```diff
@@ -1,103 +1,24 @@
...
-import org.infinispan.configuration.cache.Configurations;
-import org.infinispan.interceptors.locking.ClusteringDependentLogic;
-import org.infinispan.partitionhandling.impl.PartitionHandlingManager;
-import org.infinispan.remoting.inboundhandler.DeliverOrder;
-import org.infinispan.remoting.rpc.RpcManager;
-import org.infinispan.remoting.transport.Address;
-import org.infinispan.transaction.LockingMode;
 import org.infinispan.transaction.xa.CacheTransaction;
-import org.infinispan.transaction.xa.GlobalTransaction;
 import org.infinispan.util.logging.Log;
```
Comment (pruivo) [inline core/src/main/java/org/infinispan/transaction/impl/AbstractEnlistmentAdapter.java]: unused imports

**Comment by pruivo** [comment]
just a small comment. It looks good :)

**Comment by danberindei** [comment]
@pruivo Updated

**Comment by pruivo** [comment]
going to push...

**Comment by pruivo** [comment]
integrated! thanks @danberindei !

**Pull request closed without merging**